version.properties

```diff
@@ -1,2 +1,2 @@
-versionName=7.52-rc-2
-versionCode=9162
\ No newline at end of file
+versionName=7.52-rc-3
+versionCode=9166
```

Comment:
Do you know why the version number got bumped a few extra times? It's not a problem, I'm just curious.

Reply:
Automated upload to Google failed a few times and I had to retry.